example/basic/instance.properties

```diff
@@ -31,7 +47,7 @@ sleeper.region=eu-west-2
 sleeper.vpc=1234567890
 
 # The subnet to deploy ECS tasks to.
```

Comment:
Does this comment need updating to "A comma separated list of subnets to deploy ECS tasks to."?